lnwire/interfaces.go

```diff
@@ -0,0 +1,13 @@
+package lnwire
+
+// AnnounceSignatures is an interface that represents a message used to
+// exchange signatures of a ChannelAnnouncment message during the funding flow.
+type AnnounceSignatures interface {
```

Comment:
Would expect something around abstract sig validation as well? Or by the time we care about that, we've already done a type cast?